Automattic/wp-calypso
# Add NODE_CONFIG_ENV instead NODE_ENV

#### Changes proposed in this Pull Request

Use [NODE_CONFIG_ENV](https://github.com/lorenwest/node-config/wiki/Environment-Variables#node_config_env) for e2e tests instead NODE_ENV.

#### Testing instructions



Set up e2e tests from the beginning, this time with NODE_CONFIG_ENV, and check if everything works as expected:
- Do `unset NODE_ENV` if you already have NODE_ENV set
- `export NODE_CONFIG_ENV=<something>`
- `run npm decryptconfig` - new `local-<something>.json` file should be added to e2e/config folder
- run some tests, they should work as usual
- also, make sure that tests are passing in CircleCI
